components/softdevice/s115/doc/s115_9.0.0-3.prototype_migration-document.rst

```diff
@@ -60,6 +60,25 @@ Required changes
 - :old:`sd_power_ram_power_set`, :old:`sd_power_ram_power_get` and :old:`sd_power_ram_power_clr`
   have been removed and :new:`NRF_MEMCONF->CONTROL` should be used instead.
 
+- It is required to seed the random number generator whenever the :new:`NRF_EVT_RAND_SEED_REQUEST` event is raised.
+  Seeding is done using the new :new:`sd_rand_seed_set` API.
```

Comment:
this can be removed now https://github.com/nrfconnect/sdk-nrf-bm/blob/main/subsys/softdevice_handler/nrf_sdh_soc.c#L49

Reply:
> I'll leave that to the sdk team

why not here? Just deleting two lines, easier than communicating back and forth